kliff/trainer/kim_residuals.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any, Dict
+
+import numpy as np
+
+
+def MSE_residuals(
```

Comment:
We can port residuals at https://github.com/openkim/kliff/blob/038c990cd8980e17a2438146e3beaef09c357529/kliff/loss.py#L36 to here. 

Need to think about whether we want to support normalization and such in the residual function.

Reply:
Will return to this later, keeping the comment unresolved for reference.